### PR TITLE
Add wallet flag for allowhighfees

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

**Comment by cjepson** [comment]
It looks like a number of calls to SendRawTransaction were missed.

`grep -rsn "SendRawTransaction"
rpc/legacyrpc/methods.go:2985:  txSha, err := chainClient.SendRawTransaction(createdTx.MsgTx, false)
rpc/legacyrpc/methods.go:3066:  txSha, err := chainClient.SendRawTransaction(createdTx.MsgTx, false)
rpc/legacyrpc/methods.go:3597:                          hash, err := chainClient.SendRawTransaction(msgTx, false)
wstakemgr/stake.go:644:// SendRawTransaction sends a raw transaction using the chainSvr.
wstakemgr/stake.go:647:func (s *StakeStore) SendRawTransaction(msgTx *wire.MsgTx) (*chainhash.Hash,
wstakemgr/stake.go:655:         return s.chainSvr.SendRawTransaction(msgTx, false)
wstakemgr/stake.go:893: ssgenSha, err := s.chainSvr.SendRawTransaction(msgTx, false)
wstakemgr/stake.go:981:// submits it by SendRawTransaction. It also stores a record of it
wstakemgr/stake.go:1082:        ssrtxSha, err := s.chainSvr.SendRawTransaction(msgTx, false)`

**Comment by cjepson** [comment]
tACK On simnet, let me review the code

File: config.go [hunk 1]
```diff
@@ -93,6 +94,7 @@ type config struct {
 	PoolFees            float64 `long:"poolfees" description:"The per-ticket fee mandated by the ticket pool as a percent (e.g. 1.00 for 1.00% fee)"`
 	AddrIdxScanLen      int     `long:"addridxscanlen" description:"The width of the scan for last used addresses on wallet restore and start up (default: 750)"`
 	StakePoolColdExtKey string  `long:"stakepoolcoldextkey" description:"Enables the wallet as a stake pool with an extended key in the format of \"xpub...:index\" to derive cold wallet addresses to send fees to"`
+	AllowHighFees       bool    `long:"allowhighfees" description:"Force dcrd to allow highfees from all sendrawtransaction"`
```
Comment (cjepson) [inline config.go]: This is actually the RPC client doing this. Maybe "Force the RPC client to use the 'allowHighFees' flag when sending transactions'

File: wstakemgr/stake.go [hunk 2]
```diff
@@ -1107,7 +1107,7 @@ func (s *StakeStore) generateRevocation(blockHash *chainhash.Hash, height int64,
 func (s StakeStore) HandleWinningTicketsNtfn(blockHash *chainhash.Hash,
 	blockHeight int64,
 	tickets []*chainhash.Hash,
-	defaultVoteBits uint16) ([]*StakeNotification, error) {
+	defaultVoteBits uint16, allowHighFees bool) ([]*StakeNotification, error) {
```
Comment (cjepson) [inline wstakemgr/stake.go]: add newline after defaultVoteBits uint16,

File: wstakemgr/stake.go [hunk 3]
```diff
@@ -1163,7 +1163,7 @@ func (s StakeStore) HandleWinningTicketsNtfn(blockHash *chainhash.Hash,
 // SSRtx.
 func (s StakeStore) HandleMissedTicketsNtfn(blockHash *chainhash.Hash,
 	blockHeight int64,
-	tickets []*chainhash.Hash) ([]*StakeNotification, error) {
+	tickets []*chainhash.Hash, allowHighFees bool) ([]*StakeNotification, error) {
```
Comment (cjepson) [inline wstakemgr/stake.go]: add newline after tickets []*chainhash.Hash,

**Comment by cjepson** [comment]
Fix those issues and OK to squash and merge

**Comment by cjepson** [comment]
Also please add a detailed description in the commit

**Comment by jcvernaleo** [comment]
Looks like all was taken care of.
OK

**Pull request merged**